release.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -e
+
+FIND_CMD="find . -mindepth 2 -maxdepth 3 -print | grep Dockerfile | grep -vE '(test|.j2)'"
+BUILD_TAG="latest"
```

Comment:
Do you really want to have the latest tag? Probably, you want to control what is deployed to the cluster and avoid the latest version be updated automatically on every pod restart